docs/managed-datahub/slack/saas-slack-troubleshoot.md

```diff
@@ -0,0 +1,69 @@
+
+### Refresh the existing app installation (Recommended)
+:::note
+Whomever originally installed the Slack app will need to perform this.
+If they are unable to do this, you may need to go down the 'Install a new app' path below.
+:::
+1. Get your App Config tokens by following the first few steps outlined in the [installation guide](https://datahubproject.io/docs/managed-datahub/slack/saas-slack-setup/#step-by-step-guide). If it's showing expired tokens, feel free to delete them and create a new set.
+2. Paste them into their respective text inputs, and hit **'Re-connect'**
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/re_connect_1.png"/>
+</p>
+3. You will be re-directed to a page where you can finalize the app refresh.
+
+### Install a new app
+:::note
+If you choose to install a new app, your team will have to re-add the new bot into any private channels the old one was previously in.
+If you'd like support in getting a list of the private channels that are subscribed to Slack notifications on DataHub, please reach out to your customer success representative.
+:::
+
+1. Get your App Config tokens by following the first few steps outlined in the [installation guide](https://datahubproject.io/docs/managed-datahub/slack/saas-slack-setup/#step-by-step-guide). If it's showing expired tokens, feel free to delete them and create a new set.
+2. Paste them into their respective text inputs, and hit **'create a new installation'**
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/re_connect_1_1.png"/>
+</p>
+3. You will be re-directed to a page where you can finalize the app installation.
+4. Now to uninstall the old app, visit the **'Manage Apps'** page for your Slack workspace.
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/re_connect_2.png"/>
+</p>
+5. Find the previously installed DataHub Slack bot in the list of installed apps, and open it.
+6. Open the 'App details' page
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/re_connect_3.png"/>
+</p>
+6. Then, switch to the Configuration tab
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/re_connect_4.png"/>
+</p>
+6. Finally, scroll to the bottom to find the remove button.
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/re_connect_5.png"/>
+</p>
+
+
+## Test notification works, but not receiving notifications
+There are a few reasons why you may not receive notifications when you'd expect.
+
+### 1. Actors do not receive notifications for their actions
```

Comment:
![image](https://github.com/user-attachments/assets/e6627a3e-1de3-4500-85be-d8f09c38b718)
not really sure we need to number these headers - looks a little bit off in the toc